### ISSUE TITLE
How to plot two combined DateTime range into one chart?

### ISSUE THREAD

**Issue by uuhan** [38/plotters#239]
Hi, I love this crate for plotting. But after reading all codes under examples directory, I wonder how I can draw two sepreted but not overlaped time range into one chart without some gap between my data.

For example, I have two sets of data: 

((time1..time2), y) and ((time3..time4), y)

I want to draw them in one chart.

I can archive this by split_horizontally() the root drawing area, and draw two chart, one with y-axis and label, the other does not have y-axis and label. This "looks like" my two data sets are drawed together. But I this this is not a good way.

**Comment by 38** [comment]
Yep, that's the work around currently. But this is a nice feature to support
